src/machines/settingsMachine.ts

```diff
@@ -90,7 +90,7 @@ export const settingsMachine = setup({
         toastCallback?: () => void
         rootContext: any
       }
-    >(async ({ input }) => {
+    >(async ({ input, ...rest }) => {
```

Comment:
Nit: I don't think rest is actually used?

Reply:
Ah yup holdover from some debugging logs
```suggestion
    >(async ({ input }) => {
```